.changeset/light-coats-brush.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/shared': patch
+'@clerk/types': patch
+---
+
+Deprecate `__unstable` methods for organizations.
```

Comment:
🙃 Could you specifically reference the deprecations? `__unstable__invitationUpdate`, `__unstable__membershipUpdate`